src/renderer/renderElement.ts

```diff
@@ -822,14 +822,11 @@ const drawElementFromCanvas = (
   // Clear the nested element we appended to the DOM
 };
 
-export const renderElement = (
+export const renderElementUI = (
```

Comment:
let's move this out of `renderElement.ts` altogether, and make it just render the selection: `renderSelectionElement()`

Reply:
this is done